### PR TITLE
WIP: add contribute and project.json docs links

### PR DESCRIPTION
**Pull request by tdykstra**

### CODE REVIEW

File: index.md [hunk 1]
```diff
@@ -207,6 +207,12 @@ title: .NET Documentation
                                     </div>
                                 </div>
                             </li>
+                            <li class="fullSpan">
+                                <div class="container footer">
+                                    <p><a href="https://github.com/dotnet/Docs/blob/master/CONTRIBUTING.md">Contribute to .NET docs and samples</a></p>
+                                    <p><a href="https://docs.microsoft.com/en-us/dotnet/articles/project-json">Find Visual Studio 2015/project.json version of the documentation</a></p>
```
Comment (svick) [inline index.md]: Shouldn't this article be part of the TOC? It currently is not.

Comment (tdykstra) [inline index.md]: @svick Yes we could make a place for it in the TOC, would be a new top-level node.  @mairaw @wadepickett Thoughts?

Comment (mairaw) [inline index.md]: @tdykstra perhaps can be the last entry in the .NET Core Guide?

**Review by mairaw — APPROVED** [review]
Let's merge this one and we can take care of the TOC in a separate PR. Thanks @tdykstra!

**Comment by tdykstra** [comment]
@mairaw @svick - proposed TOC node caption "VS 2015/project.json docs" ?

**Comment by mairaw** [comment]
@tdykstra sounds good!

**Pull request merged**